packages/apidom-ls/src/config/openapi/openapi3_1/lint/webhooks--type.ts

```diff
@@ -0,0 +1,18 @@
+// OPENAPI3_1_WEBHOOKS_TYPE
```

Comment:
This should be `webhooks--values-type.ts` - which means that we're linting the type of all the values of the object and not the type of the object itself. See more [here](https://github.com/swagger-api/apidom/tree/975e12e93141897c1ae95ae372037f57a7f83832/packages/apidom-ls/src/config/openapi/components/lint).